2024-01-10 22:41:13.759423 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 22:46:36.424099 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 22:49:19.860410 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 22:51:32.399027 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 22:58:23.508268 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 23:03:27.866134 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 23:07:57.001542 - ERROR : send/read tcp data error: exceptions must derive from BaseException
2024-01-10 23:17:42.433851 - ERROR : send/read tcp data error: Traceback (most recent call last):
  File "d:\Project\Coding\PYAIBOTE\PyAibote\WebBotModel\SendTcpData.py", line 27, in SendData
    response = self.StarLoadWait(data)
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Project\Coding\PYAIBOTE\PyAibote\WebBotModel\LoadWait.py", line 40, in StarLoadWait
    if Implicit_Waiting_Throwing:
       ^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'Implicit_Waiting_Throwing' is not defined. Did you mean: 'self.Implicit_Waiting_Throwing'?

2024-01-10 23:19:29.787361 - ERROR : send/read tcp data error: Traceback (most recent call last):
  File "d:\Project\Coding\PYAIBOTE\PyAibote\WebBotModel\SendTcpData.py", line 27, in SendData
    response = self.StarLoadWait(data)
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Project\Coding\PYAIBOTE\PyAibote\WebBotModel\LoadWait.py", line 42, in StarLoadWait
    raise "Unable to find the specified element"
TypeError: exceptions must derive from BaseException

2024-01-10 23:24:20.883672 - ERROR : send/read tcp data error: Traceback (most recent call last):
  File "d:\Project\Coding\PYAIBOTE\PyAibote\WebBotModel\SendTcpData.py", line 27, in SendData
    response = self.StarLoadWait(data)
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Project\Coding\PYAIBOTE\PyAibote\WebBotModel\LoadWait.py", line 42, in StarLoadWait
    raise "Unable to find the specified element"
TypeError: exceptions must derive from BaseException

